app/models/work_package/pdf_export/work_package_list_to_pdf.rb

```diff
@@ -113,7 +113,7 @@ def render_work_packages_pdfs(work_packages, filename)
   def render_batched(work_packages, filename)
     @batches_count = work_packages.length.fdiv(@work_packages_per_batch).ceil
     batch_files = []
-    (1..@batches_count).each do |batch_index|
+    (0..(@batches_count - 1)).each do |batch_index|
```

Comment:
btw ruby also has A...B (three dots) which excludes the last value

> Ranges constructed using .. run from the beginning to the end inclusively. Those created using ... exclude the end value. When used as an iterator, ranges return each value in the sequence.

Reply:
Awesome. Thanks for the tip!